plasoscaffolder/plugins/interface.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+"""The plugin interface classes."""
+import abc
+import collections
+
+from plasoscaffolder.lib import definitions
+
+
+question = collections.namedtuple(
+    'question', ['attribute', 'prompt', 'help', 'type'])
+
+
+class ScaffolderPlugin(object):
+  """The plugin interface."""
+
+  # The name of the plugin or parser this scaffolder plugin provides.
+  PROVIDES = 'base_parser'
+  DESCRIPTION = ''
```

Comment:
As this is an interface, please add a comment here abut what DESCRIPTION should look like